app/bootstrap.php

```diff
@@ -9,6 +9,4 @@
 
 $containerBuilder = new ContainerBuilder;
 $containerBuilder->addDefinitions(__DIR__ . '/config.php');
-$container = $containerBuilder->build();
-
-return $container;
```

Comment:
Could you restore the variable? I want the demo to be as accessible as possible.